### ISSUE TITLE
Clicking outside blank username field causes Uncaught ReferenceError: formatMessage is not defined(…) - Teacher Registration

### ISSUE THREAD

**Issue by jwzimmer** [LLK/scratch-www#1098]
**TL;DR**
- Error message in console if you click outside blank username field
- No known impact on users; you can complete registration after triggering this error

**To repro:** clear the username value and click outside the fields. Now you should see the sentry error (I assume that has nothing to do with the actual teacher account functionality, just with logging the errors) and Uncaught ReferenceError: formatMessage is not defined(…).

<img width="1277" alt="screen shot 2016-12-05 at 5 25 09 pm" src="https://cloud.githubusercontent.com/assets/1320456/20905041/34b0317c-bb10-11e6-9098-78f75253eb02.png">

Discussed with @mewtaylor, currently the "This field is required" will not be translateable as-is, but the same fix to the console error will also make the message translateable.

This seems to be a bug introduced by https://github.com/LLK/scratch-www/issues/872.
